backend/remote/index.coffee

```diff
@@ -249,12 +275,25 @@ class Remote
         return callback() unless doc.remote
         remoteDoc = @createRemoteDoc doc, doc.remote
         remoteDoc._deleted = true
-        @couch.put remoteDoc, (err, removed) =>
-            if err
-                callback err, removed
-            else
+        async.waterfall [
```

Comment:
@frankrousseau you are right -> https://github.com/nono/cozy-desktop/commit/54a29cbf5bdfdaf0f9da8b0f495674cd8468660d